src/ring_theory/ideal_operations.lean

```diff
@@ -459,6 +459,112 @@ end surjective
 
 end map_and_comap
 
+section jacobson
+variables {R : Type u} [comm_ring R]
+
+/-- The jacobson radical of I is the infimum of all maximal ideals containing I. -/
+def jacobson (I : ideal R) : ideal R :=
+Inf { J : ideal R | I ≤ J ∧ is_maximal J }
```

Comment:
```suggestion
Inf {J : ideal R | I ≤ J ∧ is_maximal J}
```

Reply:
Please add `le_jacobson` stating that `I ≤ jacobson I`

Reply:
Another option is to make the definition a bit more "first order logic", as Neil Strickland suggested above.